static/app/views/performance/landing/widgets/components/selectableList.tsx

```diff
@@ -103,6 +103,17 @@ export function TimeSpentInDatabaseWidgetEmptyStateWarning() {
   );
 }
 
+export function TimeConsumingDomainsWidgetEmptyStateWarning() {
+  return (
+    <StyledEmptyStateWarning>
+      <PrimaryMessage>{t('No results found')}</PrimaryMessage>
+      <SecondaryMessage>
+        {t('Domains may not be listed due to the filters above or a low sampling rate')}
```

Comment:
What do you think about linking to docs/troubleshooting here? Would also be a good CTA to make sure people turn on their instrumentation in their SDK!

Reply:
@AbhiPrasad definitely! Linking out to "Requests" module docs 👍🏻